packages/wallets/src/features/cashier/modules/Transactions/components/TransactionsPendingRow/TransactionsPendingRow.tsx

```diff
@@ -39,109 +88,120 @@ const TransactionsCryptoRow: React.FC<TProps> = ({ transaction }) => {
+                            }}
+                        />
+                        <TransactionsPendingRowField
+                            name='Date'
+                            value={moment.unix(transaction.submit_date).format('DD MMM YYYY')}
+                            valueTextProps={{
+                                color: 'general',
+                            }}
+                        />
+                    </React.Fragment>
                 )}
-            >
-                <WalletText color='primary' size='xs'>
-                    Confirmations
-                </WalletText>
-                <WalletText align='center' color='red' size='xs' weight='bold'>
-                    {transaction.formatted_confirmations}
-                </WalletText>
-            </div>
-            <div
-                className={classNames(
-                    'wallets-transactions-pending-row__column',
-                    'wallets-transactions-pending-row__transaction-time'
+                <TransactionsPendingRowField
+                    className={{ 'wallets-transactions-pending-row__transaction-time': !isMobile }}
+                    name='Time'
+                    value={moment
+                        .unix(transaction.submit_date)
+                        .format(isMobile ? 'HH:mm:ss [GMT]' : 'DD MMM YYYY HH:mm:ss [GMT]')}
+                    valueTextProps={{
+                        color: 'general',
+                        size: isMobile ? 'xs' : '2xs',
+                        weight: isMobile ? 'bold' : 'regular',
+                    }}
+                />
+                {!isMobile && (
+                    <div className='wallets-transactions-pending-row__transaction-amount'>
+                        <WalletText
+                            align='right'
+                            color={transaction.is_deposit ? 'success' : 'red'}
+                            size='sm'
+                            weight='bold'
+                        >
+                            {transaction.is_deposit ? '+' : '-'}
+                            {transaction.formatted_amount}
+                        </WalletText>
+                    </div>
                 )}
-            >
-                <WalletText color='primary' size='xs'>
-                    Time
-                </WalletText>
-                <WalletText color='general' size='2xs'>
-                    {moment.unix(transaction.submit_date).format('DD MMM YYYY HH:mm:ss [GMT]')}
-                </WalletText>
             </div>
-            <div
-                className={classNames(
-                    'wallets-transactions-pending-row__column',
-                    'wallets-transactions-pending-row__transaction-amount'
-                )}
+            <button
+                className='wallets-transactions-pending-row__transaction-status'
+                onClick={isMobile ? onMobileStatusClick : undefined}
```

Comment:
```suggestion
                onClick={isMobile ? onMobileStatusClick : undefined}
```
can you move the isMobile check to the function instead of doing this? Doing it this way could lead to some bugs when QA resizes the browser window

Reply:
Done